x/community/client/proposal_handler.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	govclient "github.com/cosmos/cosmos-sdk/x/gov/client"
+	govrest "github.com/cosmos/cosmos-sdk/x/gov/client/rest"
+
+	"github.com/kava-labs/kava/x/community/client/cli"
+	"github.com/kava-labs/kava/x/community/types"
+)
+
+// community-pool deposit/withdraw lend proposal handlers
+var (
+	LendDepositProposalHandler = govclient.NewProposalHandler(
+		cli.NewCmdSubmitCommunityPoolLendDepositProposal,
+		notImplementedRestHandler(types.ProposalTypeCommunityPoolLendDeposit),
+	)
+	LendWithdrawProposalHandler = govclient.NewProposalHandler(
+		cli.NewCmdSubmitCommunityPoolLendWithdrawProposal,
+		notImplementedRestHandler(types.ProposalTypeCommunityPoolLendDeposit),
+	)
+)
+
+func notImplementedRestHandler(subRoute string) govclient.RESTHandlerFn {
```

Comment:
the REST handlers are deprecated, so i opted to just have them return 501s.
![](https://http.cat/501)